content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,187 @@
+- :guilabel:`Status`: the current status of the product.
+- :guilabel:`Company`: the company under which the order was placed. This only appears in a
+  multi-company database.
+
+.. image:: management/today.png
+   :align: center
+   :alt: The list that appears in the Today's Orders dashboard, with the filters and top column
+         names highlighted.
+
+.. _lunch/cancel:
+
+Cancel orders
+-------------
+
+All users can cancel an order, not just managers of the *Lunch* app.
+
+To cancel an order from a vendor, individual products **must** be canceled one at a time.
+
+On the :guilabel:`Today's Orders` dashboard, a :guilabel:`✖️ Cancel` icon is shown at the far-right
+of each product line that can be canceled. Click the :guilabel:`✖️ Cancel` icon to cancel the order
+for that individual product.
+
+.. note::
+   Only products with a red :guilabel:`Status` tag of :guilabel:`Ordered` can be canceled.
+
+.. image:: management/cancel.png
+   :align: center
+   :alt: Order lines with the cancel icon highlighted.
+
+.. _lunch/send-orders:
+
+Send Orders
+-----------
+
+The first step in managing the *Lunch* app is to send the orders to the vendors.
+
+When orders are ready to be sent, the manager responsible for sending orders **must** send the
+orders to the vendor, outside of the database (call, online order, etc.).
+
+Once orders have been placed with the vendors, click the :guilabel:`Send Orders` button that appears
+next to each vendor's name and phone number.
+
+Once sent, the :guilabel:`Send Orders` button changes to a :guilabel:`Confirm Orders` button, and
+the :guilabel:`Status` column is updated from red :guilabel:`Ordered` tags to blue :guilabel:`Sent`
+tags, indicating the order has been sent to the vendor. Users who have placed orders in the *Lunch*
+app rely on the :guilabel:`Status` tags to track their orders.
+
+.. image:: management/send.png
+   :align: center
+   :alt: A vendor's order with the X Cancel and Send Orders buttons highlighted.
+
+.. _lunch/confirm-orders:
+
+Confirm Orders
+--------------
+
+After orders have been sent to the vendor, the next step is to confirm the orders after they have
+been delivered.
+
+On the :guilabel:`Today's Orders` dashboard, click the :guilabel:`Confirm Orders` button that
+appears next to the vendor's name and phone number.
+
+Once confirmed, the :guilabel:`Confirm Orders` button disappears, and the :guilabel:`Status` column
+is updated from blue :guilabel:`Sent` tags to green :guilabel:`Received` tags, indicating the vendor
+has delivered the orders.
+
+In addition, the :guilabel:`✖️ Cancel` icon at the end of each product line changes to a
+:guilabel:`✉️ Send Notification` icon.
+
+If needed, instead of confirming all of the individual products from a vendor, individual products
+can be confirmed one at a time. To confirm an individual product, click the :guilabel:`✔️ Confirm`
+icon at the end of the individual product line. When confirming individual products with this
```

Comment:
This is a nitpick– but the term "button" seems more fitting here since these are not just an icon but a button with an icon and text.
```suggestion
can be confirmed one at a time. To confirm an individual product, click the :guilabel:`✔️ Confirm`
button at the end of the individual product line. When confirming individual products with this
```

Reply:
I was torn because it doesn't have the outline and it doesn't "look" like a button. I originally thought button but changed my mind- I am glad you convinced me otherwise =)

Reply:
I updated all of them- the send notification, cancel, etc- they wre all referenced as icons.